Launcher.py

```diff
@@ -1,5 +1,5 @@
 """
-Archipelago launcher for bundled app.
+Archipelago launcher for bundled MDApp.
```

Comment:
was this just a find-replace? i'm not sure exactly what the text is trying to say but i don't think it needs to specify that it uses MDApp specifically

Reply:
Yeah. The commit "lmao" was me fixing the other ones that were wrong, but this one seemed fine. Not married to it though.

Reply:
just rewrite the entire docstring instead :p